enforcer-rules/pom.xml

```diff
@@ -128,4 +128,27 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-invoker-plugin</artifactId>
+        <version>3.2.1</version>
+        <configuration>
+          <cloneProjectsTo>${project.build.directory}/it</cloneProjectsTo>
+          <localRepositoryPath>${project.build.directory}/local-repo</localRepositoryPath>
+          <postBuildHookScript>verify</postBuildHookScript>
```

Comment:
Document for the parameters: http://maven.apache.org/plugins/maven-invoker-plugin/run-mojo.html#cloneProjectsTo